composer.json

```diff
@@ -35,17 +39,19 @@
         "drupal/field_group": "1.x-dev"
     },
     "require-dev": {
-        "behat/mink": "~1.7",
-        "behat/mink-goutte-driver": "~1.2",
-        "jcalderonzumba/gastonjs": "~1.0.2",
-        "jcalderonzumba/mink-phantomjs-driver": "~0.3.1",
-        "mikey179/vfsStream": "~1.2",
-        "phpunit/phpunit": "~4.8",
-        "symfony/css-selector": "~2.8",
-        "drupal/devel": "1.x-dev"
+        "behat/behat": "^3",
+        "behat/mink-goutte-driver": "^1.2",
+        "jcalderonzumba/gastonjs": "^1.0.2",
+        "jcalderonzumba/mink-phantomjs-driver": "^0.3.1",
+        "mikey179/vfsStream": "^1.2",
+        "phpunit/phpunit": "^5",
+        "symfony/css-selector": "^2.8",
+        "drupal/devel": "1.x-dev",
+        "geerlingguy/drupal-vm": "dev-master"
     },
     "conflict": {
-        "drupal/drupal": "*"
+        "drupal/drupal": "*",
+        "drupal/link-link": "*"
```

Comment:
This is due to https://www.drupal.org/node/2795709, which still seems to crop up in annoying instances.